radix-engine-common/src/crypto/ed25519/mod.rs

```diff
@@ -0,0 +1,9 @@
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
For the purposes of the toolkit, this should be a feature flag and not dependent on the architecture. This is because the toolkit is compiled to WASM and requires all of these cryptographic types.

This also applies to the other curves too.

Reply:
First of all thanks for valuable feedback!
When introducing the BLS I paid special attention to not narrow down the scope and functionality of the RE.
Before this change the cryptographic features were located in `transaction` crate, which was not built for scrypto (`wasm32` target). 
As the matter of fact ATM it is not possible to build `secp2561k1` for WASM (some deps) - this is the main reason I've decided to not include cryprographic features for `wasm32` and this is why I decided to use target architecture switches. 

Regarding RE cryptographic types - these are being compiled.

So these switches are not excluding any functionality that was enabled before the change.
Maybe I will just put some comment in the code to remind us to remove them once `wasm32` target finally works?

Reply:
After a discussion with @0xOmarA agreed that it cryptographic primitives will be enabled under some feature gate. Radix Engine Toolkit is needing them for all supported targets (for `wasm32` target too - apparently it is possible to compile `secp2561k1` for `wasm32`).
Above change will be introduced in separate PR.

Reply:
After second discussion we agreed that no need to feature gate cryptographic primitives. We just need to make sure all compiles for `wasm32` target. If yes then it will be enabled by default.